flask_project/flaskproject.py

```diff
@@ -0,0 +1,22 @@
+from flask import Flask, escape, request, render_template, url_for, flash, redirect
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = '93590f0e3dc7099d1f75f2607779ad4f'
```

Comment:
Can we use environmental variables here instead to manage our keys
Python does allow for key management and access using os like so: 

```
import os
os.environ['SOME_SECRET_KEY'] 
```

You can then add the key via your CLI using 
`export SOME_SECRET_KEY=_<THIS_IS_THE_SECRET_KEY>_`